.github/workflows/README.md

```diff
@@ -0,0 +1,59 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> A simple project to learn about best practices in using Github and Git flow
+
+![screenshot](./app_screenshot.PNG)
+
+Additional description about the project and its features.
+
+## Built With
+
+-HTML & CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
+
+👤 **Mnemba Chambuya**
+
+- GitHub: [@githubhandle](https://github.com/mnekx)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/mnemba-chambuya)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly remove the `README.md` file from the `.github/workflows` directory. `README.md` file should be in the root directory just like the `index .html` and `styles.css`

See how it will look:
<img width="537" alt="Loom_9HtxpqAIVJ" src="https://user-images.githubusercontent.com/81468675/148058940-619591a4-2ea9-4e3d-9250-f6e94227d383.png">